docs/developers/contributing/simple-module.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: standardheadmd
+title: '- Nucleus Developers'
+header: Creating a Simple module
+---
+
+## Explanation
+A simple module is a module which does not have any configuration whatsoever. You can add commands, listeners, and tasks to a module. In 
+order to begin creating a module you must first locate the src/main/java/io/github/nucleuspowered/nucleus/modules package. Next a new
```

Comment:
The package is `io.github.nucleuspowered.nucleus.modules`. The _path in the repo_ is this.